src/invidious/views/watch.ecr

```diff
@@ -1,12 +1,12 @@
 <% content_for "header" do %>
 <meta name="thumbnail" content="<%= thumbnail %>">
-<meta name="description" content="<%= video.short_description %>">
+<meta name="description" content="<%= video.short_description.gsub("\"", "&quot;") %>">
```

Comment:
Should be `HTML.escape(video.short_description)`, similar to `og:title`, `twitter:title`, etc.